client/my-sites/checkout/checkout-thank-you/jetpack-checkout-siteless-thank-you.tsx

```diff
@@ -65,13 +70,19 @@ const JetpackCheckoutSitelessThankYou: FC< Props > = ( { productSlug, receiptId
 				recordTracksEvent( 'calypso_siteless_checkout_submit_website_address', {
 					product_slug: productSlug,
 					site_url: siteUrl,
+					receipt_id: receiptId,
 				} )
 			);
 			dispatch( requestUpdateJetpackCheckoutSupportTicket( siteUrl, receiptId ) );
-			// On successful response redirect to schedule 15min Happiness support page? (Calendly?)
 		}
 	}, [ siteInput, dispatch, productSlug, receiptId ] );
 
+	useEffect( () => {
+		if ( supportTicketStatus && supportTicketStatus === 'success' ) {
+			page.redirect( `/checkout/jetpack/thank-you-completed/no-site/${ productSlug }` );
```

Comment:
Minor: Something subtle that I've run into before is that `page.redirect()` _replaces_ the current URL with the new one in the browser history. If you don't want that and just want a normal redirect, it's better to use `page()`.